recipes/cajun-jsonapi/all/conanfile.py

```diff
@@ -1,49 +1,89 @@
-from conans import ConanFile, tools
+from conan import ConanFile
+from conan.tools.build import check_min_cppstd
+from conan.tools.files import get, copy, load, save
+from conan.tools.scm import Version
 import os
 
+required_conan_version = ">=1.53.0"
+
+
 class CajunJsonApiConan(ConanFile):
-    name = 'cajun-jsonapi'
-    description = 'CAJUN* is a C++ API for the JSON object interchange format.'
+    name = "cajun-jsonapi"
+    description = "CAJUN* is a C++ API for the JSON object interchange format."
     topics = ("conan", "cajun", "json")
     url = "https://github.com/conan-io/conan-center-index"
     homepage = "https://github.com/cajun-jsonapi/cajun-jsonapi"
     license = "BSD-3-Clause"
     settings = "compiler"
```

Comment:
```suggestion
    settings = "os", "arch", "compiler", "build_type"
```
Please, keep full settings then manage it on the package ID.